CoVAR-app/src/app/(pages)/evaluate/page.tsx

```diff
@@ -1,59 +1,108 @@
-    if (file) {
-      setSelectedFile(file);
-    }
-  };
+  useEffect(() => {
+    const fetchAssignedUsers = async () => {
+      try {
+        const token = localStorage.getItem('accessToken');
+        const response = await axios.get('/api/users/assigned_clients', {
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+        });
+        setUsers(response.data);
+        console.log('Assigned users:', response.data);
+      } catch (err) {
+        console.error('Error fetching assigned users:', err);
+      }
+    };
 
-  const handleSubmitFile = () => {
-    console.log('File submitted:', selectedFile);
+    const fetchAssignedOrganizations = async () => {
+      try {
+        const token = localStorage.getItem('accessToken');
+        const response = await axios.get('/api/users/assigned_organizations', {
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+        });
+        setOrganizations(response.data);
+        console.log('Assigned organizations:', response.data);
+      } catch (err) {
+        console.error('Error fetching assigned organizations:', err);
+      }
+    };
+
+    fetchAssignedUsers();
+    fetchAssignedOrganizations();
+  }, []);
+
+  const handleUserButtonClick = (user: User) => {
+    router.push(`/evaluate/user/${user.username}`);
+  };
 
+  const handleOrganizationButtonClick = (organization: Organization) => {
+    router.push(`/evaluate/organization/${organization.name}`);
   };
 
   return (
-      <Box sx={mainContentStyles}>
-        <Paper sx={uploadBoxStyles}>
-          <Typography variant="h6">
-            Upload a Vulnerability Assessment
-          </Typography>
-          <input
-            type="file"
-            accept=".pdf,.csv"
-            onChange={handleFileChange}
-            style={{ display: 'none' }}
-            id="file-upload"
-          />
-          <label htmlFor="file-upload">
-            <Button variant="contained" component="span" sx={uploadButtonStyles}>
-              Upload File
-            </Button>
-          </label>
-          {selectedFile && (
-            <>
-              <Typography variant="body2">
-                Selected File: {selectedFile.name}
-              </Typography>
-              <Button
-                variant="contained"
-                sx={{ ...uploadButtonStyles, marginTop: 2 }}
-                onClick={handleSubmitFile}
-              >
-                Submit File
-              </Button>
-            </>
-          )}
-        </Paper>
-      </Box>
+    <Box sx={mainContentStyles}>
+      <Typography variant="h6" sx={{ marginTop: 4 }}>
+        Assigned Clients and Organizations
+      </Typography>
+      <Paper elevation={3} sx={{ padding: 2, marginTop: 2 }}>
+        <List>
+          {users.map(user => (
+            <ListItem key={user.user_id} sx={{ marginBottom: 1, padding: 1, borderRadius: 1, boxShadow: 1 }}>
```

Comment:
Can you display some text informing the user if they have no clients assigned to them